### DISCUSSION TITLE
WooCommerce Subscriptions

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5051] [Working on Features]
Integrate Fluxstore with the WooCommerce Subscriptions plugin is: https://woocommerce.com/products/woocommerce-subscriptions/

**Comment by ameenbadri7** [comment]
This feature is most wanted.

**Comment by QaisAljedyani** [comment]
Definitely needed and it will be a great selling point for new adapters of the software

**Comment by DavideM12** [comment]
Integrate with fluxnews also

**Comment by myu98198** [comment]
Please add subscription feature its most requested
